packages/web/src/hooks/starknet/FullScreenProvider/context.ts

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
could we move this out of the starknet folder and into the base hook folder, and combine the provider and hooks into a single file? otherwise lgtm